department-of-veterans-affairs/vets-website
# My VA - Appointment card timezone offset bug

## Summary

This PR:
- updates how we format the time on the AppointmentsCard component on My VA so that Veteran will still see appointment time rather than their system time
- updates unit tests and some helper functions

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/73088
- original [Slack thread](https://dsva.slack.com/archives/C909ZG2BB/p1704481343168139)

## Testing done

- locally, visually
- tested by changing location (and therefore time) on both system and browser

## To test

- run My VA locally 
  ```
    yarn watch --env entry=dashboard
    yarn mock-api --responses src/applications/personalization/dashboard/mocks/server.js
  ```
- follow these steps to change your timezone: https://stackoverflow.com/a/60008052
- check the time in the Next appointment card under Health care
- make sure time is 6:30 a.m. MT, regardless of time/location set in browser

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

![Screenshot 2024-01-09 at 1 40 35 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/8542413/9ffb1ad7-4b6a-4deb-8c0b-de9510afb5fa)   
![Screenshot 2024-01-09 at 1 40 28 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/8542413/867a4b33-71b2-4560-a0c5-b2ccbdcd9557)    
![Screenshot 2024-01-09 at 1 40 19 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/8542413/346efffc-3604-4813-857c-6d306eec340d) 

## What areas of the site does it impact?

My VA - Health care - Next appointments card

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication
